MAUI/ImageEditor/text.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
+title: Text Annotation in .NET MAUI ImageEditor control | Syncfusion
+description: Learn here all about the text annotation feature of Syncfusion .NET MAUI ImageEditor(SfImageEditor) control.
+platform: .NET MAUI
+control: SfImageEditor
+documentation: ug
+---
+
+# Text Annotation in .NET MAUI ImageEditor (SfImageEditor)
+
+The ImageEditor control in .NET MAUI allows you to add text with customizable settings.
+
+{% tabs %}
+{% highlight C# tabtitle="C#" %}
+
+    imageEditor.AddText("Good morning");
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customize text settings
+
+The text annotation can be customized using common annotation customization properties:
+
+* `Bounds`: Specifies the bounds of the annotation view.
```

Comment:
AllowDrag: Enables or disables the dragging of text annotation.
AllowResize: Enables or disables the resizing of shape annotation.
Bounds: Specifies the bounds of the text view. You can position the text wherever you want on the image. The value of the text bounds should fall between 0 and 1.
Opacity: Specifies the opacity of shape annotation. This value ranges from 0 to 1.
`RotationAngle`: Specifies the initial rotation angle of the text. The value should fall between 0 to 360.
* `IsEditable`: Enables or disables text editing.
* `IsRotatable`: Enables or disables the text rotation.
* `TextAlignment`: Specifies the start, center, or end alignment of the text.
* `TextStyle`: Change the text appearance such as `TextColor`, `FontSize`, `FontAttributes`, and `FontFamily` in the image editor using the `ImageEditorTextSettings.TextStyle` property.

Reply:
Updated a content